### ISSUE TITLE
No mention of 2.4.3 systems for Cross-Site Scripting security issues in newest security patches

### ISSUE THREAD

**Issue by Hurrpadurr** [magento/magento2#36294]
### Summary (*)

In the recent releases of Magneto Updates (2.4.4-p2 and 2.4.5-p1) a critical Cross-Site Scripting was addressed. There is no Security patch for 2.4.3 systems in spite of support for those not ending until November.

I realize this is not a technical issue and thus violates the guidelines but the message board moves very slowly and this might be a pressing issue for some.

### Proposed solution

Add information on whether or not 2.4.3 Systems are affected by the same cross-site scripting vulnerabilites as their derivatives.

---
Please provide [Severity](https://devdocs.magento.com/guides/v2.4/contributor-guide/contributing.html#backlog) assessment for the Issue as Reporter. This information will help during Confirmation and Issue triage processes.

- [ ] Severity: **S0** _- Affects critical data or functionality and leaves users with no workaround._
- [ ] Severity: **S1** _- Affects critical data or functionality and forces users to employ a workaround._
- [ ] Severity: **S2** _- Affects non-critical data or functionality and forces users to employ a workaround._
- [x] Severity: **S3** _- Affects non-critical data or functionality and does not force users to employ a workaround._
- [ ] Severity: **S4** _- Affects aesthetics, professional look and feel, “quality” or “usability”._

**Comment by CanBeer** [comment]
@Hurrpadurr We are currently on Magento 2.4.3-p2 EE, can you tell us if we are also affected by these security vulnerabilities?

Link of the report : https://helpx.adobe.com/security/products/magento/apsb22-48.html

I am a little confused for the generic formulation "2.4.4-p1 and earlier versions".

Can you provide us with the patches to be made or the files/commits concerned by the corrections in version 2.4.4-p2 or 2.4.5-p1?

Thank you very much

**Comment by t-eichmann** [comment]
Same here, we are on the latest security patch for 2.4.3, but no mention for that in the security bulletin. Would be nice to at least have a link to the commits that fixed the issue, so that we can check ourselves.

**Comment by ZIPITRO** [comment]
Same here, would be good to know which commits where done that fixed them, thanks!

**Comment by engcom-November** [comment]
Hi @Hurrpadurr , Thank you for reporting and collaboration.
Magento Commerce versions 2.4.2 (and earlier), and 2.3.7 (and earlier) are affected by a stored cross-site scripting vulnerability and its mention can be found in the release notes of Magento [2.4.2](https://devdocs.magento.com/guides/v2.4/release-notes/open-source-2-4-2.html) and [2.3.7.](https://devdocs.magento.com/guides/v2.3/release-notes/open-source-2-3-7.html) And mentioned vulnerabilities are not applicable since Magento 2.4.2-P2 (2.4.3 Systems should not be affected by the same cross-site scripting vulnerabilities) hence are not mentioned in 2.4.3 and later release versions of Magento. If you have found a vulnerability in one of the latest current Magento versions, please report it through HackerOne as its not recommended to post vulnerability steps in public github repos.
Thank you.

**Comment by t-eichmann** [comment]
The link you provided for the patch gives me a 404 not found, would be nice if you could update the link or provides the commit IDs.

**Comment by mlaurense** [comment]
Getting a 404 on the link to the patch

**Comment by t-eichmann** [comment]
> Getting a 404 on the link to the patch

Try this one: https://experienceleague.adobe.com/docs/commerce-knowledge-base/kb/troubleshooting/known-issues-patches-attached/0-day-vulnerability-patch.html?lang=en

**Comment by mlaurense** [comment]
I think i found it here: https://experienceleague.adobe.com/docs/commerce-knowledge-base/kb/troubleshooting/known-issues-patches-attached/adobe-commerce-2.4.3-p2-2.4.5-security-hotfix-for-cve-2022-35698.html?lang=en

**Comment by t-eichmann** [comment]
> > > Getting a 404 on the link to the patch
> > 
> > 
> > Try this one: https://experienceleague.adobe.com/docs/commerce-knowledge-base/kb/troubleshooting/known-issues-patches-attached/0-day-vulnerability-patch.html?lang=en
> 
> That is regarding APSB22-12, not APSB22-48

Sorry, my mistake, didn't pay as much attention as I should have.

**Comment by hostep** [comment]
Yeah indeed, I see why they wanted to change the url (since the version numbers it contained were incorrect), but they really should have added a 301 redirect from the old url to the new one. Even basic CMS systems (like the one from Magento itself) can handle such things, no idea why that Adobe ExperienceWhatever can't do it ...

**Comment by nathanjosiah** [comment]
The URL was changed and we found out that the platform somehow doesn't support URL redirects. 
Here is the new one [https://experienceleague.adobe.com/docs/commerce-knowledge-base/kb/troubleshooting/known-i[…]-2.4.3-p2-2.4.5-security-hotfix-for-cve-2022-35698.html](https://experienceleague.adobe.com/docs/commerce-knowledge-base/kb/troubleshooting/known-issues-patches-attached/adobe-commerce-2.4.3-p2-2.4.5-security-hotfix-for-cve-2022-35698.html)
Regarding 2.4.3, there is a hotfix for 2.4.3-p2 and 2.4.3-p3 available in the article. 2.4.3 and 2.4.3-p1 are not affected as long as you have all previously existing hotfixes applied.
